content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,206 @@ aliases:
+
+**Note**: This example uses a [`COUNT` metric type][1], but the count of custom metrics would have been the same with a [`GAUGE`][2] or a [`RATE`][3] metric type.
+
+Suppose you’re submitting a metric, `request.Count`, from two hosts (A,B), which counts the number of endpoint requests. You’re submitting this metric with two tag keys:
+
+* `endpoint`  that can take the value `X` or `Y`
+* `status` that can take the value `200` or `400`
+
+Let’s assume that in your data, endpoint `X` is supported by both hosts, but fails only on `host:B`, and requests to `endpoint:Y` are always successful and only appears on `host:B` as shown below:
+
+{{< img src="account_management/billing/custom_metrics/custom_metrics_host.png" alt="Custom metrics host" responsive="true" style="width:80%;">}}
+
+There are then **4 Custom metrics**, one for each unique combinations of metric name, host, and tag values for your `request.Count` metric. The custom metrics are listed below:
+
+| Metric Name     | Tag Values                           |
+| --------------- | ------------------------------------ |
+| `request.Count` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Count` | `host:B`, `endpoint:Y`, `status:200` |
+
+[1]: /developers/metrics/types/?tab=count
+[2]: /developers/metrics/types/?tab=gauge
+[3]: /developers/metrics/types/?tab=rate
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. [Learn more about HISTOGRAM metric type][1].
+
+Suppose you have a histogram metric measuring `request.Latency` for two endpoints `X` and `Y`  on a single host:
+
+|| INSERT IMAGE  ||
+
+The number of unique tag value combinations submitted for a `HISTOGRAM` metric with this tagging scheme is 2, but the Agent generates [5 custom metrics][1] for each of the original 2 unique tag value combinations to account for each Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. This results in a total of 10 custom metrics for this `HISTOGRAM` metric: :
+
+| Metric Name       | Tag Values             | Aggregations                            |
+| ----------------- | ---------------------- | --------------------------------------- |
+| `request.Latency` | `host:A`, `endpoint:X` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:A`, `endpoint:Y` | `count`, `sum`, `min`, `max`, and `avg` |
+
+[1]: /developers/metrics/types/?tab=histogram
+{{% /tab %}}
+{{% tab "Distribution" %}}
+
+**A DISTRIBUTION metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the server-side aggregations  `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
```

Comment:
The number of custom metrics from a [DISTRIBUTION metric][1] is 5x the number of custom metrics emitted from a COUNT/RATE/GAUGE. That is, for each unique combination of metric name and tag values, 5 custom metrics are generated to support the Distribution's server-side aggregations (count:, sum:, min:, max: and avg:)